src/main/java/frc/robot/Robot.java

```diff
@@ -94,4 +153,157 @@ public void teleopPeriodic() {
   @Override
   public void testPeriodic() {
   }
+
+  /**
+   * This function drives the robot at a certain speed.
+   */
+  public void driveSpeed() {
+    if (!colorWheelMode) {
+      double rawAxis1 = driveController.getRawAxis(1);
+      double rawAxis4 = driveController.getRawAxis(4);
+
+      // Enabling slow speed toggle on the joystick for drive speed
+      if (driveController.getRawAxis(2) > 0.5) {
+        slowToggle = true;
+      } else {
+        slowToggle = false;
+      }
+
+      // Enabling high speed toggle on the joystick for drive speed
+      if (driveController.getRawAxis(3) > 0.5) {
+        highToggle = true;
+      } else {
+        highToggle = false;
+      }
```

Comment:
Some thoughts on this:
- This code uses an analog input for a digital output; that is, a stick (or trigger?) value is being used for a toggle. This is a little confusing for the driver because it could give the impression that the more it is held, the faster. I think it would be better to find a button for these, if possible.
- Why are the `slowToggle` and `highToggle` booleans necessary? It seems you could just check for that and set the robot drive speed in the same code.

Reply:
I could understand the confusion, yes. I will look into assigning a button to these toggles. The `slowToggle` and `highToggle` booleans are not necessary, however I added them for personal structural preferences, but they can be removed.

Reply:
Yeah, I would rather those two booleans be removed from the class.